BUILD.gn

```diff
@@ -0,0 +1,530 @@
+# Copyright (c) 2016 Intel Corporation. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import("//build/config/features.gni")
+import("//build/config/sanitizers/sanitizers.gni")
+import("//build/config/ui.gni")
+import("//build_overrides/v8.gni")
+import("//media/media_options.gni")
+import("//third_party/widevine/cdm/widevine.gni")
+import("//third_party/ffmpeg/ffmpeg_options.gni")
+import("//xwalk/build/version.gni")
+
+config("xwalk_runtime_config") {
+  # TODO(rakuco): Remove this once we stop supporting Ubuntu Precise or
+  # default to use_sysroot==1. This is only required because Precise's dbus
+  # 1.4.18 causes
+  # <https://bugs.chromium.org/p/chromium/issues/detail?id=263960#c6>.
+  if (is_clang && !use_sysroot) {
+    cflags = [ "-Wno-reserved-user-defined-literal" ]
+  }
+}
+
+source_set("xwalk_runtime") {
```

Comment:
The entire ordering of the entries in this target does not match GN's style guide's recommendations.